WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -241,7 +237,7 @@ - (RemoteBlogSettings *)remoteBlogSettingFromJSONDictionary:(NSDictionary *)json
     
     remoteSettings.name = [json stringForKey:BlogRemoteNameKey];
     remoteSettings.desc = [json stringForKey:BlogRemoteDescriptionKey];
-    remoteSettings.defaultCategory = [rawSettings numberForKey:BlogRemoteDefaultCategoryKey] ?: @(PostCategoryUncategorized);
+    remoteSettings.defaultCategory = [rawSettings numberForKey:BlogRemoteDefaultCategoryKey] ?: @(1);
```

Comment:
@koke I was trying to remove any dependency of the Model objects from the Remote classes. The constant is being defined and also used on the Model object. 
I can redefine it here for the remote class do you think it's better?